Dockerfile

```diff
@@ -1,20 +1,19 @@
-FROM navikt/node-express:14
+FROM navikt/node-express:16
 
-ARG NODE_ENV
```

Comment:
Vi trenger egentlig ikke lenger `NODE_ENV`, da NAIS allerede eksponerer en variabel for oss som heter `NAIS_CLUSTER_NAME`